app/models/organization.rb

```diff
@@ -14,7 +14,7 @@ class Organization < ApplicationRecord
   has_many :contributors, dependent: :destroy
   has_many :requests, dependent: :destroy
   has_many :notifications_as_mentioned, class_name: 'ActivityNotification', dependent: :destroy
-  has_many :messages, through: :requests
+  has_many :messages, dependent: :destroy
```

Comment:
Why didn't we do this earlier?


Is the connection between request<->organization now given through messages? :thinking:

Reply:
I think it wasn't there before because the dependent option is set on the request since it went through the request. It would be redundant to have it in both places.

Quite possibly, but I don't know if this PR is the place to drill all these down. We still could benefit from doing this with the contributor.organization after we decided that a contributor can only ever belong to one organization.